pom.xml

```diff
@@ -1183,6 +1019,263 @@
+                </goals>
+                <configuration>
+                  <includes>
+                    <include>**/*FT.java</include>
+                  </includes>
+                  <excludes>
+                    <exclude>**/*Test.java</exclude>
+                    <exclude>**/*IT.java</exclude>
+                    <exclude>**/*FfT.java</exclude>
+                  </excludes>
+                  <argLine>${jacoco.agent.arg}</argLine>
+                  <skipTests>${skipFTs}</skipTests>
+                  <systemPropertyVariables>
+                    <fester.logs.output>${seeLogsFT}</fester.logs.output>
+                    <fester.s3.bucket>${fester.s3.bucket}</fester.s3.bucket>
+                    <fester.placeholder.url>${fester.placeholder.url}</fester.placeholder.url>
+                    <fester.container.tag>${docker.registry.account}${project.artifactId}:${project.version}</fester.container.tag>
+                    <festerize.version>${festerize.version}</festerize.version>
+                    <jdwp.host.port>${jdwp.host.port}</jdwp.host.port>
+                    <jdwp.client.config>${jdwp.client.config}</jdwp.client.config>
+                  </systemPropertyVariables>
+                </configuration>
+              </execution>
+              <execution>
+                <id>integration-tests</id>
+                <phase>integration-test</phase>
+                <goals>
+                  <goal>test</goal>
+                </goals>
+                <configuration>
+                  <includes>
+                    <include>**/*IT.java</include>
+                  </includes>
+                  <excludes>
+                    <exclude>**/*FT.java</exclude>
+                    <exclude>**/*Test.java</exclude>
+                    <exclude>**/*FfT.java</exclude>
+                  </excludes>
+                  <argLine>${jacoco.agent.arg}</argLine>
+                  <skipTests>${skipITs}</skipTests>
+                  <systemPropertyVariables>
+                    <fester.logs.output>${seeLogsIT}</fester.logs.output>
+                    <fester.s3.region>${fester.s3.region}</fester.s3.region>
+                    <fester.s3.bucket>${fester.s3.bucket}</fester.s3.bucket>
+                    <fester.s3.access_key>${fester.s3.access_key}</fester.s3.access_key>
+                    <fester.s3.secret_key>${fester.s3.secret_key}</fester.s3.secret_key>
+                    <fester.s3.endpoint>${fester.s3.endpoint}</fester.s3.endpoint>
+                    <fester.container.tag>${docker.registry.account}${project.artifactId}:${project.version}</fester.container.tag>
+                    <jdwp.host.port>${jdwp.host.port}</jdwp.host.port>
+                    <jdwp.client.config>${jdwp.client.config}</jdwp.client.config>
+                  </systemPropertyVariables>
+                </configuration>
+              </execution>
+            </executions>
+          </plugin>
+        </plugins>
+      </build>
+    </profile>
```

Comment:
I'm curious why this change

Reply:
Moving this code into a default profile (that's activated by the absence of a `-Dversions` argument, ensures these plugins don't run when we're just running the versions profile (that profile just uses a standard ubuntu image so the tests would fail against it).

Reply:
ah I see, that makes a lot of sense